src/frequenz/sdk/timeseries/formula_engine/_formula_generators/_simple_power_formula.py

```diff
@@ -0,0 +1,69 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Formula generator from component graph."""
+
+from frequenz.client.microgrid import ComponentCategory, ComponentMetricId
+
+from ....microgrid import connection_manager
+from ..._quantities import Power
+from .._formula_engine import FormulaEngine
+from ._formula_generator import FormulaGenerator
+
+
+class SimplePowerFormula(FormulaGenerator[Power]):
+    """Formula generator from component graph for calculating sum of Power.
+
+    The producer power is calculated by summing up the power of requested components.
+
+    Raises:
+        RuntimeError: If no components are defined in the config or if any
+            component is not found in the component graph.
+    """
```

Comment:
Not sure if the second line belongs here.

```suggestion
    """Formula generator from component graph for calculating sum of Power.

    Raises:
        RuntimeError: If no components are defined in the config or if any
            component is not found in the component graph.
    """

Reply:
Done